english/chuck-norris/chucknorris.md

```diff
@@ -0,0 +1,5 @@
+You pull out your cell phone and text Chuck Norris to ask what he would do.
+
+Does he tell you to:
+
+[Tie the fork in knots?](https://github.com/mongoose1729/create-your-own-adventure/blob/master/english/fork-knots/melt-with-fork.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.